src/treemap/imp.rs

```diff
@@ -161,16 +160,16 @@ impl Treemap {
     /// treemap.add(120);
     /// treemap.add(1000);
     ///
-    /// assert_eq!(treemap.maximum(), 1000);
-    /// assert_eq!(empty_treemap.maximum(), 0);
+    /// assert_eq!(treemap.maximum(), Some(1000));
+    /// assert_eq!(empty_treemap.maximum(), None);
     /// ```
-    pub fn maximum(&self) -> u64 {
+    pub fn maximum(&self) -> Option<u64> {
         self.map
             .iter()
+            .rev()
```

Comment:
Why reverse the iterator direction here?

Reply:
I think the node with big numbers are at the end of iteration?

Reply:
That's true - makes sense!